x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/convert/ToIpTests.java

```diff
@@ -182,8 +183,10 @@ private static String readEvaluator() {
     }
 
     private static String stringEvaluator(ToIp.LeadingZeros leadingZeros) {
+        if (leadingZeros == null) {
```

Comment:
As this was here before, and in jdk21 the `case null` works (right?), do we need this change?